pkg/io4edge/channel.go

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright © 2021 Ci4Rail GmbH <engineering@ci4rail.com>
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io4edge
+
+import (
+	"errors"
+	"time"
+
+	"github.com/ci4rail/io4edge-client-go/pkg/io4edge/transport"
+	"google.golang.org/protobuf/proto"
+)
+
+// Channel represents a io4edge channel
+type Channel struct {
+	ms transport.MsgStream
+}
+
+// NewChannel creates a new channel using the transport mechanism in t
+func NewChannel(ms transport.MsgStream) (*Channel, error) {
+	return &Channel{ms: ms}, nil
+}
+
+// Close closes the message stream
```

Comment:
remove "Close "

Reply:
don't remove "Close ", better rephrase: "// Close terminates the message stream"